pagerduty/resource_pagerduty_event_orchestration.go

```diff
@@ -97,7 +102,7 @@ func expandOrchestrationTeam(v interface{}) *pagerduty.EventOrchestrationObject
 	var team *pagerduty.EventOrchestrationObject
 	t := v.([]interface{})[0].(map[string]interface{})
 	team = &pagerduty.EventOrchestrationObject{
-		ID: t["id"].(string),
+		ID: stringTypeToStringPtr(t["id"].(string)),
```

Comment:
Team ID is now a string pointer because, unlike strings, pointers can be assigned a `nil` value, so we need to convert it to string.